keras/engine/training.py

```diff
@@ -2427,7 +2427,7 @@ def evaluate_generator(self, generator, steps=None,
                     averages.append(np.average([out[i] for out in all_outs],
                                                weights=batch_sizes))
                 else:
-                    averages.append(all_outs[-1][i])
+                    averages.append(float(all_outs[-1][i]))
```

Comment:
Why `float` (not `np.float64`)?
When TensorBoard callback is used with stateful metrics it raise
```
File "...\keras\callbacks.py", line 942, in on_epoch_end
    summary_value.simple_value = value.item()
AttributeError: 'float' object has no attribute 'item'
```
`np.float64` works correctly due to existence of `item()` class method.